requirements.txt

```diff
@@ -6,3 +6,4 @@ build==0.3.0
 twine==3.3.0
 wheel==0.36.2
 setuptools==53.0.0
```

Comment:
Fine for now. Let's talk about this offline - last I checked, pip's algorithm for solving `requirements.txt` constraints isn't stable. I think we'll eventually want to use `pip-tools` to produce a lockfile for reproducible builds, which will also let us get out of `==` pins here.

Reply:
https://github.com/envoyproxy/protoc-gen-validate/issues/630